1.8/usage/service-discovery/marathon-lb/index.md

```diff
@@ -2,21 +2,21 @@
 post_title: Marathon-LB
 ---
 
-marathon-lb is based on HAProxy, a rapid proxy and load balancer. HAProxy provides proxying and load balancing for TCP and HTTP based applications, with features such as SSL support, HTTP compression, health checking, Lua scripting and more. marathon-lb subscribes to Marathon’s event bus and updates the HAProxy configuration in real time.
+Marathon-lb is based on HAProxy, a rapid proxy and load balancer. HAProxy provides proxying and load balancing for TCP and HTTP based applications, with features such as SSL support, HTTP compression, health checking, Lua scripting and more. Marathon-lb subscribes to Marathon’s event bus and updates the HAProxy configuration in real time.
 
 Up to date documentation for MLB can be found on the GitHub page.
```

Comment:
`Up-to-date documentation for Marathon-lb can be found on the GitHub page.` (otherwise I'm thinking... Major League Baseball)